### PR TITLE
Motions 2019 02 cwg 1

### LINKED ISSUE #2681: [2019-02 CWG Motion 1] P1358R0 Core Language Working Group "ready" Issues (Kona 2019)
[P1358R0](https://wg21.link/p1358r0)

### LINKED ISSUE #2716: [range.iota, range.adaptors] View's synopses missing cross-references to iterator and sentinel members
Per https://github.com/cplusplus/draft/pull/2475#issuecomment-466289153, these are bugs.

### PR DESCRIPTION
**Pull request by burblebee**
Fixes #2681.

### CODE REVIEW

File: source/exceptions.tex [hunk 1]
```diff
@@ -365,12 +365,12 @@
...
 to a handler,
-destructors are invoked by a process, specified in this subclause, called
-\defn{stack unwinding}.
+objects with automatic storage duration are destroyed by a process,
+specified in this subclause, called \defn{stack unwinding}.
 
 \pnum
-The destructor is invoked for each automatic object of class type constructed,
-but not yet destroyed,
+Each object with automatic storage duration is destroyed if it has been
```
Comment (anerokhi) [inline source/exceptions.tex]: Does not look like semantic line break (https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines#source-code-formatting)

**Comment by zygoloid** [comment]
Force-pushed with some fixes:

CWG2267: Use list-init syntax for example (fixing CWG wording) -- fix confirmed with CWG
CWG2309: Fixed bad indexing markup in [stmt.stmt]
CWG2310: [expr.static.cast]p11: CWG added "complete" in the wrong place, p12: "of a complete class D" doesn't make sense because we already introduced D earlier in the sentence -- fix confirmed with CWG
CWG2318: Improved formatting for P'_i
CWG2330: Was missing third change
CWG2331: *Change dropped; returned to CWG for redrafting*. Wording is incorrect and intended to fix an editorial issue (my fault; my analysis in #1169 is wrong)
CWG2332: Editorially fixed incorrect note and clarified normative wording -- fix confirmed with CWG
CWG2360: Was missing second addition

**Comment by burblebee** [comment]
@zygoloid thanks so much for fixing my misses!  would be great to point out what those misses are in the future tho so I can learn from them!  might that be possible?

**Comment by johelegp** [comment]
Looks like you're looking for the [force-pushed](https://github.com/cplusplus/draft/compare/79b94bd02172006d04973ab96e30adbb1f45598f..27f5ad594ce343b658d673728f4020e65f1c09ca) link above his comment.

**Pull request merged**